docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1026,3 +1028,113 @@ function su_humsci_profile_update_9714() {
     $role->save();
   }
 }
+
+/**
+ * Remove any existing hs_webform paragraph entities.
+ */
+function su_humsci_profile_update_9715($sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $paragraph_machine_name = 'hs_webform';
+
+  if (empty($sandbox['ids'])) {
+    $sandbox['ids'] = $paragraph_storage->getQuery()
+      ->condition('type', $paragraph_machine_name)
+      ->accessCheck(FALSE)
+      ->execute();
+    $sandbox['total'] = count($sandbox['ids']);
+  }
+  $paragraph_ids = array_splice($sandbox['ids'], 0, 50);
+
+  foreach ($paragraph_storage->loadMultiple($paragraph_ids) as $paragraph) {
+    $paragraph->delete();
+  }
+
+  $sandbox['#finished'] = count($sandbox['ids']) ? 1 - count($sandbox['ids']) / $sandbox['total'] : 1;
+}
+
+/**
+ * Delete hs_webform paragraph type and remove webform config.
+ */
+function su_humsci_profile_update_9716() {
+  $paragraph_machine_name = 'hs_webform';
+  // Delete the field and field storage on the paragraph.
+  $fields = \Drupal::entityTypeManager()
+    ->getStorage('field_config')
+    ->loadByProperties(['bundle' => $paragraph_machine_name]);
+
+  if ($fields) {
+    foreach ($fields as $field) {
+      /** @var Drupal\field\Entity $field */
+
+      // Delete the field configuration (instance) from the paragraph type.
+      $field->delete();
+    }
+  }
+
+  // Delete the field_hs_webform field storage.
+  $field_storage = FieldStorageConfig::load('field_hs_webform');
+  if ($field_storage) {
+    $field_storage->delete();
+  }
+
+  // Delete the paragraph itself.
+  $paragraph_type = ParagraphsType::load($paragraph_machine_name);
+  if ($paragraph_type) {
+    $paragraph_type->delete();
+  }
+
+  // Remove hs_webform from config ignored configs.
+  $config_storage = \Drupal::service('config.storage.sync');
+
+  $configs_to_change = [
+    'node.hs_basic_page.field_hs_page_components' => 'node.field_hs_page_components',
+    'paragraph.hs_collection.field_hs_collection_items' => 'paragraph.field_hs_collection_items',
+    'node.hs_private_page.field_hs_priv_page_components' => 'node.field_hs_priv_page_components',
+  ];
+
+  /** @var \Drupal\Core\Entity\EntityStorageInterface $field_config_storage */
+  $field_config_storage = \Drupal::service('entity_type.manager')->getStorage('field_config');
+
+  /** @var \Drupal\Core\Entity\EntityStorageInterface $fieldConfigStorage */
+  $field_storage = \Drupal::service('entity_type.manager')->getStorage('field_storage_config');
+
+  foreach ($configs_to_change as $field_config => $field_storage_config) {
```

Comment:
Is this necessary if the configs are to be synced after the db update?

Reply:
Config for `field.field.node.hs_basic_page.field_hs_page_components` and `field.field.node.hs_private_page.field_hs_priv_page_components` is [partially overridden](https://github.com/SU-HSDO/suhumsci/blob/416ecc8ccfedb3e759e7c91e129a92b8acd7e363/config/default/config_ignore.settings.yml#L23), so it won't be synced when config is imported. The config for `field_hs_collection_items` is not overridden, so I removed it, but noticed that `field.field.node.hs_basic_page.field_hs_page_hero` was missing, so I added it.

Reply:
Yes, they are partially overridden for a reason. Doing this, re-syncs the configuration with the file system, essentially ignoring why that is partially ignored. This can have negative results.